paragon-cli.js

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+
+const inquirer = require('inquirer');
+const childProcess = require('child_process');
+
+const themes = [
+  { name: '@edx/brand-openedx', value: '' },
+  { name: '@edx/brand-edx.org', value: 'npm:@edx/brand-edx.org@latest' },
+];
+
+function promptUserForTheme() {
+  return inquirer.prompt([
+    {
+      type: 'list',
+      name: 'theme',
+      message: 'Choose a theme to install:',
+      choices: themes.map((theme) => theme.name),
+    },
+  ]);
+}
+
+function installTheme(theme) {
+  const installCommand = `npm install "@edx/brand@${theme}" --no-save`;
```

Comment:
[suggestion] This can totally be deferred, but an idea: make the `--no-save` configurable such that the command can be run where this `npm install` _does_ save. The default should be `--no-save`, though. I'm thinking of a possible use case where the Paragon CLI could be used in infrastructure/deployment code to install the `@edx/brand` aliases rather than the infrastructure/deployment code needing to the the `npm install` themselves (might be easier to simply run `paragon install-theme` during the build+deploy process 🤷‍♂️).

Reply:
Yes, I think it is good idea, maybe flag `-d`, `--deploy` or something will solve the problem and will make it clear for the user. Also I created issue for your three comments that won't be part of this PR. https://github.com/openedx/paragon/issues/2483

Reply:
Thanks for creating the issue!